x-pack/plugin/core/src/main/resources/monitoring-beats.json

```diff
@@ -52,6 +52,9 @@
                     "name": {
                       "type": "keyword"
                     },
```

Comment:
One thought I had was to add `"copy_to": "hostname"` here, for the case of an older-version Beat (sending in `name`) working with a newer-version ES. Thoughts, reviewers?

Reply:
Both fields can exist in newer version with potentially different values. So better not.

Reply:
Oh, I didn't realize that. I thought `hostname` was a **replacement** for the `name` field. If that's not the case, then it makes sense not to copy from name -> hostname.

Reply:
You have a point here. For the monitoring data reporting it is actually a replacement / renaming. I was thinking of the data shipped by Beats.